MobileCenter/MobileCenter/MSWrapperSdk.h

```diff
@@ -31,15 +31,6 @@
  */
 @property(nonatomic, copy, readonly) NSString *liveUpdatePackageHash;
 
-/**
- * Is equal to another wrapper SDK
- *
- * @param object Wrapper SDK
- *
- * @return Return YES if equal and NO if not equal
- */
-- (BOOL)isEqual:(id)object;
-
 - (instancetype)initWithWrapperSdkVersion:(NSString *)wrapperSdkVersion
```

Comment:
We have many methods without docs. We have to address that, can be a separate task.